src/ui/map.js

```diff
@@ -2626,6 +2632,7 @@ class Map extends Camera {
             if (!this.isMoving() && this.loaded()) {
                 this.fire(new Event('idle'));
                 if (this._isInitialLoad) {
+                    // Following line is billing related code. Do not change. See LICENSE.txt
```

Comment:
Maybe lift this statement outside the conditional? `s/line/block/`? No strong feeling here, as long as the intent is clear.

Reply:
Edit: Now that I re-read, the intent is clear. Kindly disregard.